packages/patternlab/styleguide/source/assets/js/modules/accordions.js

```diff
@@ -54,7 +54,7 @@ export default (function (window,document,$,undefined) {
       $toggleLink.removeClass('ma__collapsible-content__toggle-all--collapsed');
     }
 
-    $('.js-accordion').each(function(index){
+    $('.ma__collapsible-content--extended .js-accordion').each(function(index){
```

Comment:
I don't see this class in the DOM `ma__collapsible-content--extended`? https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22657_accordion_adjustments/?p=organisms-collapsible-content-as-steps
Where is this used?

Reply:
ok I see it now. It's in the wrapper div for https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22657_accordion_adjustments/?p=organisms-collapsible-content-extended